composure.sh

```diff
@@ -329,9 +329,10 @@ glossary ()
   group 'composure'
 
   typeset targetgroup=${1:-}
-  typeset maxwidth=$(_longest_function_name_length | awk '{print $1 + 5}')
+  typeset functionlist="$(_typeset_functions)"
+  typeset maxwidth=$(_longest_function_name_length "$functionlist" | awk '{print $1 + 5}')
 
-  for func in $(_typeset_functions); do
+  for func in $functionlist; do
```

Comment:
This works for both bash and zsh:

    list="a b c"
    for i in $(echo $list); do echo $i; done

im going to create an additional commit